frontend/src/scenes/surveys/surveyViewViz.tsx

```diff
@@ -119,20 +129,86 @@ export function Summary({
     surveyUserStats: SurveyUserStats
     surveyUserStatsLoading: boolean
 }): JSX.Element {
-    if (!surveyUserStats) {
-        return <></>
-    }
-
     return (
-        <div className="mb-4">
+        <div className="mb-4 mt-2">
             {surveyUserStatsLoading ? (
                 <LemonTable dataSource={[]} columns={[]} loading={true} />
             ) : (
                 <>
-                    <UsersCount surveyUserStats={surveyUserStats} />
-                    <UsersStackedBar surveyUserStats={surveyUserStats} />
+                    {!surveyUserStats ? null : (
+                        <>
+                            <UsersCount surveyUserStats={surveyUserStats} />
+                            <UsersStackedBar surveyUserStats={surveyUserStats} />
+                        </>
+                    )}
                 </>
             )}
         </div>
     )
 }
+
+export function RatingQuestionBarChart({
+    questionIndex,
+    question,
```

Comment:
same idea as above, you don't need to pass in both questionIndex and question. Just the index is enough to uniquely identify the question.

And within the component you can get the question via `useValues(surveyLogic)`.

If this were in a backend context, I'd be all for trying to keep functions as pure as possible since there's no shared state. But here, there's already a lot of shared state, such that we run into the risk of mismatching the index and the question / having to always take care they are in sync when they represent the same data.